static/app/utils/analytics/profilingAnalyticsEvents.tsx

```diff
@@ -0,0 +1,15 @@
+type ProfilingPrefix = 'profiling';
+type ProfilingViews =
+  | 'landing'
+  | 'transaction_summary'
+  | 'profile_summary'
+  | 'flamegraph';
+
+type EventKey = `${ProfilingPrefix}_views.${ProfilingViews}`;
```

Comment:
@JonasBa what about typing the payload?

Reply:
I checked other events and just went with `string` type as most seem to do that. Given that it's a human readable string, I think it makes sense - typing both the key and values seems counterintuitive. Fwiw, I think the right way to do that would be to generate those human readable values using a helper fn, that way we have consistent values in amplitude

Reply:
@JonasBa I'm talking about the payload which might have extrainformation: https://github.com/getsentry/sentry/blob/master/static/app/utils/analytics/dashboardsAnalyticsEvents.tsx#L4-L9

> . Fwiw, I think the right way to do that would be to generate those human readable values using a helper fn, that way we have consistent values in amplitude

Yea, the fact we have these two different names is kind of silly but that decision pre-dates when I joined. That being said, I actually prefer explicitly writing out the human-readable names because rather than a mapping function because then someone can go in and search the code base for a particular event which often happens. Consistent naming is less important than the discoverability of the underlying events.

Reply:
@JonasBa got it, yes